app/src/main/res/values/strings.xml

```diff
@@ -5,23 +5,33 @@
   <string name="info_screen_name">Info</string>
   <string name="required">Required</string>
   <string name="optional">Optional</string>
+  <string name="read_more">Read more</string>
+  <string name="here">here</string>
+  <string name="result">Result</string>
+  <string name="none">None</string>
   <string name="details">Details</string>
 
   <!-- Section titles -->
   <string name="section_title_sdk_initialization">SDK Initialization</string>
 
   <!-- SDK Initialization -->
+  <string name="sdk_initialization_description">This method initializes the SDK and connects the device with Access. It has to be called before using any of the SDK features. It returns a list of removed user profiles (in case any profile got removed on Access), so the App can update it\'s state.</string>
```

Comment:
This method initializes the SDK and connects the device with the server. It has to be called before using any of the SDK features. It returns a list of removed user profiles in case any profile got removed on the server. 

I would not use a name of the backend server as it can change over time; token server -> access -> ???